frontend-react/src/components/FileHandlers/FileHandlerMessaging.tsx

```diff
@@ -31,11 +31,10 @@ export const FileSuccessDisplay = ({
     const { destinations, timestamp, reportId } = extendedMetadata;
     const destinationsDisplay =
         destinations || "There are no known recipients at this time.";
-
     return (
         <>
             <StaticAlert type={"success"} heading={heading} message={message} />
-            <div>
+            <div data-testid="sucess-display">
```

Comment:
We should shy away from `data-testid` -- [see this](https://youtu.be/mSYJ5Ehbo3U)

> This library encourages your applications to be more accessible and allows you to get your tests closer to using your components the way a user will, which allows your tests to give you more confidence that your application will work when a real user uses it.

Reply:
looks like I wasn't targeting this anywhere, probably added it and realized later that I didn't need it but forgot to remove. Thanks for the catch. One interesting point in this conversation is that the trussworks components we use are littered with `test-id`s: https://github.com/trussworks/react-uswds/blob/a4501ddc24c12bbe3cf1279edbcf650f9dccd412/src/components/forms/FileInput/FileInput.tsx#L153

Related testing library best practices question: since it seems like in some cases (such as `<Label>`) where there is no role assignment, your only real option is to find elements by text, which is sort of awkward when what you want to assert is that a certain element has a certain text. I addressed that just targeting an element by the expected text, (as I did here https://github.com/CDCgov/prime-reportstream/pull/6266/files#diff-af78dbee2b26ac35a63755e5665a4afb277e6a04ea9fbc35c246ad1f744954a7R33) but that seems a little awkward and backwards. Am I doing that right, or is there another way?

Reply:
also, thanks for the inspirational (?) video. are you familiar with [Brendan and Cory](https://www.youtube.com/watch?v=IXai3D4FqXY )?

Reply:
😂 Wrong link, my bad! Meant to link to the RTL home page section where that quote was from.

YES, I'm not a fan of Trussworks for that. We had something of an internal debate at Fearless once about testing methodology -- literal (by the code) vs practical (like an end user interacting with it). Both serve well, and clearly there's a need for both. I like the Label example, with it not having a defined role. I think to better understand which makes more sense in your linked example is whether the `findByText` returns the element you need to test. If not, there might be another way to query, or it might warrant a test-id! There's times you get this weird `couldn't find element by text -- maybe it's split into different elements` error from RTL. There's workarounds for that kinda thing, but that's where I've used the test-id in the past.